servers/src/grin/server.rs

```diff
@@ -80,38 +80,31 @@ pub struct Server {
 }
 
 impl Server {
-	/// Instantiates and starts a new server. Optionally takes a callback
-	/// for the server to send an ARC copy of itself, to allow another process
-	/// to poll info about the server status
-	pub fn start<F>(config: ServerConfig, mut info_callback: F) -> Result<(), Error>
-	where
-		F: FnMut(Server),
-	{
-		let mining_config = config.stratum_mining_config.clone();
-		let enable_test_miner = config.run_test_miner;
-		let test_miner_wallet_url = config.test_miner_wallet_url.clone();
-		let serv = Server::new(config)?;
+	/// Start ancillary services
+	pub fn start_ancillary(&self) -> Result<(), Error> {
```

Comment:
I changed the name because I moved the Server::new(config) out to the calling function, this is where the server is actually started.
And the reason I did that is because the Receiver is not clonable so has to be passed as another parameter in the callback which did not look very pretty.
I can do it this way if you want though, I don't really mind. Will change it so you can see.